### ISSUE TITLE
Freeze Transformer Weight

### ISSUE THREAD

**Issue by ivokun** [EleutherAI/gpt-neo#259]
Can we finetune the model with Transformer layers' weight freeze (only train the embeddings and softmax)?

**Comment by StellaAthena** [comment]
That is not currently supported by this codebase but is easy enough to do using either the HuggingFace library or by writing a small wrapper for this codebase.
